README.md

```diff
@@ -164,7 +172,7 @@ end
 ### Properties
 - `action` - `:install` to install `kubectl`, `:remove` to uninstall `kubectl`.
 - `version` - The desired version of `kubectl`.
-- `binary_path` - Application path (if empty, default:/usr/local/bin/kubectl)
+- `binary_path` - Application path (if empty, default:/usr/local/bin)
```

Comment:
default:/usr... => default: /usr... (we need a white space after the semicolon character)